api/task.go

```diff
@@ -74,12 +74,35 @@ const (
 	TaskDatabaseBackup TaskType = "bb.task.database.backup"
 	// TaskDatabaseRestore is the task type for restoring databases.
 	TaskDatabaseRestore TaskType = "bb.task.database.restore"
+	// TaskDatabasePITRRestore is the task type for restoring databases using PITR.
+	TaskDatabasePITRRestore TaskType = "bb.task.database.pitr.restore"
+	// TaskDatabasePITRCutover is the task type for swapping the pitr and original database.
+	TaskDatabasePITRCutover TaskType = "bb.task.database.pitr.cutover"
+	// TaskDatabasePITRDelete is the task type fo deleting the original database.
+	TaskDatabasePITRDelete TaskType = "bb.task.database.pitr.delete"
 )
 
 // These payload types are only used when marshalling to the json format for saving into the database.
 // So we annotate with json tag using camelCase naming which is consistent with normal
 // json naming convention
 
+// TaskDatabasePITRRestorePayload is the task payload for database PITR restore.
+type TaskDatabasePITRRestorePayload struct {
+	// The project owning the database.
+	ProjectID     int    `json:"projectId,omitempty"`
+	SchemaVersion string `json:"schemaVersion,omitempty"`
```

Comment:
Add a comment on what's SchemaVersion? Also not sure why it's needed.

Reply:
Removed.